cmd/containerd-shim-runhcs-v1/stats/stats.proto

```diff
@@ -3,21 +3,45 @@ syntax = "proto3";
 package containerd.runhcs.stats.v1;
 
 import weak "gogoproto/gogo.proto";
+import "google/protobuf/timestamp.proto";
+import "github.com/containerd/cgroups/stats/v1/metrics.proto";
 
 option go_package = "github.com/Microsoft/hcsshim/cmd/containerd-shim-runhcs-v1/stats;stats";
 
 message Statistics {
 	oneof container {
 		WindowsContainerStatistics windows = 1;
-		LinuxContainerStatistics linux = 2;
+		io.containerd.cgroups.v1.Metrics linux = 2;
 	}
 	VirtualMachineStatistics vm = 3 [(gogoproto.customname) = "VM"];
 }
 
 message WindowsContainerStatistics {
+	google.protobuf.Timestamp timestamp = 1 [(gogoproto.stdtime) = true, (gogoproto.nullable) = false];
+	google.protobuf.Timestamp container_start_time = 2 [(gogoproto.stdtime) = true, (gogoproto.nullable) = false];
+	uint64 uptime_ns = 3 [(gogoproto.customname) = "UptimeNS"];
+	WindowsContainerProcessorStatistics processor = 4;
+	WindowsContainerMemoryStatistics memory = 5;
+	WindowsContainerStorageStatistics storage = 6;
 }
 
-message LinuxContainerStatistics {
+message WindowsContainerProcessorStatistics {
+	uint64 total_runtime_ns = 1 [(gogoproto.customname) = "TotalRuntimeNS"];
```

Comment:
Why did you change the type here?

Reply:
The internal hcs types are uint64, but the code generation tool used to create our hcs schema file has a limitation and will not produce the correct type.

Reply:
Oh weird. Your right ok this is fine.